srcpkgs/swapspace/files/swapspace/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -e
+
+[ -d /var/lib/swapspace ] || install -d -m0700 /var/lib/swapspace
```

Comment:
Use `make_dirs` in template instead

Reply:
This dir is only needed during service runtime, keeping an empty dir at all times seems odd

Reply:
There has been a discussion about switching to lowercase only package names, but it would require changes in xbps and some people were against it, so we are staying with the old policy - respect how upstream names it.

Reply:
Don't you run that service all the time?

Reply:
Good point, also I see that `/var/lib/*` dirs are not typically created in runit scripts